frontend/src/bundles/chats/components/chat-search-bar/chat-search-bar.tsx

```diff
@@ -0,0 +1,42 @@
+import { type FormEvent } from 'react';
+
+import { Input } from '~/bundles/common/components/components.js';
+import { useAppForm, useCallback } from '~/bundles/common/hooks/hooks.js';
+
+const ChatSearchBar = (): JSX.Element => {
+    const { control, errors, handleSubmit } = useAppForm({
+        defaultValues: { search: '' },
+        mode: 'onChange',
+    });
+
+    const handleSearchChats = useCallback((): void => {}, []);
+
+    const handleFormChange = useCallback(
+        (event_: React.BaseSyntheticEvent): void => {
+            void handleSubmit(handleSearchChats)(event_);
+        },
+        [handleSearchChats, handleSubmit],
+    );
+
+    const handleFormSubmit = useCallback(
+        (event_: FormEvent<HTMLFormElement>): void => {
+            event_.preventDefault();
+        },
+        [],
+    );
+
+    return (
+        <form onChange={handleFormChange} onSubmit={handleFormSubmit}>
+            <Input
+                placeholder="Find a friend"
+                className="h-[3.75rem] justify-center"
+                name="search"
+                label=""
+                control={control}
+                errors={errors}
+            />
+        </form>
```

Comment:
Is it necessary to have a form here? Maybe just input with onChange?

Reply:
Since I added textarea in input component I think it'll be much easier to handle change event in form.

Reply:
I think if I add onChange to input then there will be a lot of unnecessary logic to implement in useAppForm.